File: 2025/November-7-2025/validPalindrome.py
import re
from typing import List

s = "Was it a car or a cat I saw?"


def validPalindrome(s: str) -> bool: 
  word = re.sub(r'[^A-Za-z0-9]', "", s).lower()
  return word == word[::-1]

print(validPalindrome(s))

File: 2025/September-29-2025/validPalindrome.js
// Input: s = "Was it a car or a cat I saw?"

// Output: true

var s = "Was it a car or a cat I saw?"


const validPalindrome = (s) => { 

  let check = s.replace(/[^a-z0-9]/gi, '').toLowerCase()

  return check === check.split('').reverse().join('')
}

console.log(validPalindrome(s))